syntax = "proto3";

package hello;

message HelloMessage{
  string name = 1;
  string msg = 2;
}

message ReplyMessage{
  string reply_msg = 1;
}

service HelloService{
  rpc HelloServer (stream HelloMessage) returns (stream ReplyMessage) {}
}